src/compiler/crystal/syntax/lexer.cr

```diff
@@ -493,6 +493,8 @@ module Crystal
             case char
             when '\\'
               case char = next_char
+              when 'a'
+                io << "\x07"
```

Comment:
I'd keep `"\u{7}"` format.

Reply:
Yeah basically you have to do whatever the `\b` addition does. New source code must still compile with the old compiler.